android-components/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -74,7 +73,7 @@ class CustomTabsToolbarFeature(
     private var initialized: Boolean = false
     private val titleObserver = CustomTabSessionTitleObserver(toolbar)
     private val context get() = toolbar.context
-    internal var readableColor = Color.WHITE
+    internal var readableColor = -1
```

Comment:
you might be able to do a `lateinit internal var` instead of using `-1`

Reply:
I considered this, but it would mean also making it nullable - which is not what we expect. However, the fact we both thought about this is perhaps indicative of a code smell. I've updated this to pass the `readableColor` value in as a param instead of having the class property